.cargo/audit.toml

```diff
@@ -4,6 +4,9 @@ ignore = [
     "RUSTSEC-2020-0159", # `chrono` localtime_r segfault
 ]
 
+show = ["unmaintained", "unsound"]
+fatal = ["critical"]
```

Comment:
I don't see where these option names come from, the [advisories config section](https://github.com/rustsec/rustsec/blob/87ba35d955503bec39c2c518b1b468ec4d0258b4/cargo-audit/src/config.rs#L96) looks like it does not include them.